services/console-proxy/server/src/com/cloud/consoleproxy/ConsoleProxySecureServerFactoryImpl.java

```diff
@@ -38,6 +38,11 @@
     private static final Logger s_logger = Logger.getLogger(ConsoleProxySecureServerFactoryImpl.class);
 
     private SSLContext sslContext = null;
+    private static String[] SSL_PROTOCOLS = { "TLSv1", "TLSv1.1", "TLSv1.2" };
+    private static String[] SSL_CIPHERS = { "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA", "TLS_DHE_RSA_WITH_AES_128_GCM_SHA256", "TLS_DHE_RSA_WITH_AES_128_CBC_SHA256",
+            "TLS_RSA_WITH_AES_128_GCM_SHA256", "TLS_RSA_WITH_AES_128_CBC_SHA256", "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256", "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
```

Comment:
I will check this approach and return with a feedback soon. Thanks for the hint @rhtyd !